testsuite/smoke-test/src/aptos_cli/validator.rs

```diff
@@ -1005,7 +1005,8 @@ async fn test_join_and_leave_validator() {
     gas_used += get_gas(
         cli.withdraw_stake(validator_cli_index, withdraw_stake)
             .await
-            .unwrap(),
+            .unwrap()
+            .remove(0),
```

Comment:
This should be all of the responses not just the first one, but you can fix this after

Reply:
In this particular test, there's only one stake pool.